### DISCUSSION TITLE
How to show complex Prop Types in ArgsTable (TS, React, Addon Doc, CSF)

### DISCUSSION THREAD

**Discussion by dkleber89** [storybookjs/storybook#15182] [Help]
Is there a possible way to show complex data type props in Docs ArgsTable?

At this Time i only see in ArgsTable the name of the complex data type not what is contained in the datatype.

Ist there a way to show the props of the complex datatype maybe with a custom configuration?

Thanks for help

**Comment by dmackerman** [comment]
As far as I can tell, there are numerous issues with ArgsTable that are unresolved. I can't even get a simple one to render.
